File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/javadoc/comment/direct/Java/Go/s545439898_commented.go
package main

import (
	"fmt"
)

func main() {
	var count int
	fmt.Scan(&count)
	numbers := make([]int, count)
	for i := 0; i < count; i++ {
		fmt.Scan(&numbers[i])
	}
	for i := count - 1; i >= 0; i-- {
		fmt.Printf("%d", numbers[i])
		if i != 0 {
			fmt.Print(" ")
		}
	}
	fmt.Println()
}

// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/multiline/comment/direct/Java/Python/s545439898_commented.py
import sys

count = int(input())
giving_numbers = []

# Read numbers
for _ in range(count):
    giving_numbers.append(int(input()))

# Reversing numbers
for i in range(count - 1, -1, -1):
    print(giving_numbers[i], end=" " if i > 0 else "")

print()
# End of Code
